slack-server-integration-common/src/main/resources/token/token-status.js

```diff
@@ -64,6 +65,13 @@ require([
             });
         }
 
+        function isInIssueView() {
+            return $('#issue-content').length;
+        }
+        function isSlackPanelVisible() {
+            return $("#slack-issue-panel").length;
+        }
```

Comment:
I also thought about that duplicated code but I didn't find out how to do it because the two JS files are located in separate apps. Do you have an idea?

Agree on the prevention of ambiguitites.

Reply:
Oh, I see what you mean. This file is located in the common module, while other 2 ones are in Jira plugin module.
Ok, then some duplication is acceptable.

Reply:
Could you also add a comment for these functions mentioning that they are expected to return `true` only when they are used in Jira plugin?
This file is also included into [Confuence plugin descriptor](https://github.com/atlassian-labs/atlassian-slack-integration-server/blob/master/confluence-slack-integration/confluence-slack-server-integration-plugin/src/main/resources/atlassian-plugin.xml#L490) and will be called there as well. The functions will return `false` there always, because there will be no elements with Jira-plugin-specific IDs on a Confluence page. It's better to keep this detail saved in a comment, then to learn it from debugging years later.